CMakeLists.txt

```diff
@@ -49,7 +49,7 @@ find_package(VecCore ${VecCore_VERSION} REQUIRED COMPONENTS ${VecCore_BACKEND})
 message(STATUS "Using VecCore version ${VecCore_VERSION}")
 
 # Find VecGeom geometry headers library
-set(VecGeom_VERSION 1.1.16)
+set(VecGeom_VERSION 1.1.17)
```

Comment:
Can you please also update the documentation in `README.md`?

Reply:
done